src/org/opensolaris/opengrok/analysis/Ctags.java

```diff
@@ -237,6 +238,28 @@ private void initialize() throws IOException {
             command.add("--regex-pascal=/^(uses|interface|implementation)$/\\1/s,Section/");
             command.add("--regex-pascal=/^unit[[:space:]]+([a-zA-Z0-9_<>, ]+)[;(]/\\1/u,unit/");
             
+            // PowerShell
```

Comment:
this will be a bit hard to merge once we drop exuberant ctags (I was kinda expecting an if which will decide what to do for exctags and what for uctags), but it's an option
and congratulations to working travis! :)